documentation/operators/src/faq/mixnodes-faq.md

```diff
@@ -0,0 +1,57 @@
+
+For more detailed calculation, read our blog post [Nym Token Economics update](https://blog.nymtech.net/nym-token-economics-update-fedff0ed5267). More info on staking can be found [here](https://blog.nymtech.net/staking-in-nym-introducing-mainnet-mixmining-f9bb1cbc7c36). And [here](https://blog.nymtech.net/want-to-stake-in-nym-here-is-how-to-choose-a-mix-node-to-delegate-nym-to-c3b862add165) is more info on how to choose a mix node for delegation. And finally an [update](https://blog.nymtech.net/quarterly-token-economic-parameter-update-b2862948710f) on token economics from July 2023.
+
+### Which VPS providers would you recommend?
+
+Consider in which jurisdiction you reside and where do you want to run a mix node. Do you want to pay by crypto or not and what are the other important particularities for your case? We always recommend operators to try to chose smaller and decentralized VPS providers over the most known ones controlling a majority of the internet. We receive some good feedback on these: Hertzner, DigitalOcean, Linode, Ghandi and Exoscale. Do your own research and share with the community.
+
+<!---### Why is a mix node setup on a self-hosted machine so tricky?--->
+
+### What's the Sphinx packet size?
+
+The sizes are shown in the configs [here](https://github.com/nymtech/nym/blob/1ba6444e722e7757f1175a296bed6e31e25b8db8/common/nymsphinx/params/src/packet_sizes.rs#L12) (default is the one clients use, the others are for research purposes, not to be used in production as this would fragment the anonymity set). More info can be found [here](https://github.com/nymtech/nym/blob/4844ac953a12b29fa27688609ec193f1d560c996/common/nymsphinx/anonymous-replies/src/reply_surb.rs#L80).
+
+### Why a mix node and a gateway cannot be bond to the same wallet?
+
+Becauase of the way the smart contract works we keep it one-node one-address at the moment.
+
+### Which nodes are the most needed to be setup to strengthen Nym infrastructure and which ones bring rewards?
+
+Right now only mix nodes are rewarded. We're working on gateway and service payments. Gateways are the weak link right now due mostly to lack of incentivisation. Services like Network Requesters are obviously the most necessary for people to start using the platform, and we're working on smart contracts to allow for people to start advertising them the same way they do mix nodes.
+
+### Are mixnodes whitelisted?
+
+Nope, anyone can run a mix node. Purely reliant on the node's reputation (self stake + delegations) & routing score.
+
+## Validators and tokens
```

Comment:
Shouldn't this be on its own 'validators & tokens' FAQ page maybe? Just noticed this, missed it first time around..

Reply:
I can work out one like that in October. There are more points I have saved to it.